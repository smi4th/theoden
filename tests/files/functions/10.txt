fonction a() {
    return 1;
}
fonction b() {
    return a();
}
v = b();