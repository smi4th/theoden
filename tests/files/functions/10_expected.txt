---TEST OUTPUT---
---MEMORY---
v = ('int', 1)
---FUNCTIONS---
a = ('empty', ('linst', ('return', ('int', 1))))
b = ('empty', ('linst', ('return', ('call', 'a', 'empty'))))
